src/zenml/zen_stores/schemas/base_schemas.py

```diff
@@ -26,6 +29,20 @@ class BaseSchema(SQLModel):
     created: datetime = Field(default_factory=datetime.utcnow)
     updated: datetime = Field(default_factory=datetime.utcnow)
 
+    def to_model(
```

Comment:
I went through the SQL zen store code as well as a few schemas and it looks like 1. the `include_resources` arg is never set to True in `to_model` calls and 2. the `include_resources` arg is never actually used in schema implementations. What did you have in mind as a follow-up step for this flag ?

Reply:
as we slowly migrate to the new paradigm, the include resources flag will be used in the places where they should be included. 
I fixed all the places where include_resources was falsely used